DatadogInternal/Sources/Utils/DDError.swift

```diff
@@ -24,11 +24,13 @@ public struct DDError: Equatable, Codable, PassthroughAnyCodable {
     public let type: String
     public let message: String
     public let stack: String
+    public let sourceType: String
 
-    public init(type: String, message: String, stack: String) {
+    public init(type: String, message: String, stack: String, sourceType: String = "ios") {
```

Comment:
I wonder if we should care if it is iOS, iPadOS, tvOS or VisionOS here? 🤔

Reply:
It's fine to use `ios` as default here 👍 `source_type` is an [enum](https://github.com/DataDog/rum-events-format/blob/master/schemas/rum/error-schema.json#L112L125) with fixed values. It is used for symbolication and `ios` is an umbrella for apple platforms.

Reply:
Yeah, `source_type` here is more "what's the format of the crash report" rather than what's the OS.